src/mbgl/renderer/render_layer.cpp

```diff
@@ -186,7 +186,8 @@ std::size_t RenderLayer::removeAllDrawables() {
 }
 
 void RenderLayer::updateRenderTileIDs() {
-    const auto oldMap = std::move(renderTileIDs);
+    const auto oldMap = renderTileIDs;
```

Comment:
this copies the map instead of moving. there's a performance penalty.

Reply:
I can change it, but then I have to re-initialize the data member, otherwise it is used after being moved.